source/isaaclab/isaaclab/terrains/utils.py

```diff
@@ -132,6 +136,63 @@ def create_prim_from_mesh(prim_path: str, mesh: trimesh.Trimesh, **kwargs):
         sim_utils.bind_physics_material(prim.GetPrimPath(), f"{prim_path}/physicsMaterial")
 
 
+def create_mesh_from_prim(prim_path: str, stage: Usd.Stage | None = None) -> trimesh.Trimesh:
```

Comment:
```suggestion
def create_trimesh_from_prim(prim_path: str, stage: Usd.Stage | None = None) -> trimesh.Trimesh:
```
NIT: might name it trimesh for explicitness

Reply:
Sounds good on parity!